### ISSUE TITLE
Requirements for custom SVGs to work properly

### ISSUE THREAD

**Issue by YannMiro** [FlorianWoelki/obsidian-iconize#272]
Not sure if it's necessarily a bug, but it just seems very hard to create a SVG that is compatible with your plugin. I understand that this specific feature is not stable, but if I copy-paste an image from a pre-existing pack that in my custom folder it will work fine, so I think this might be a problem with the software used.

I used Photoshop and Inkscape. The only important thing I do is make work path / trace bitmap aside from removing backgrounds and zeroing RGB in the subject area when necessary (most are simple, nothing complex or colourful). 

Yet it shows no reaction when I change the whole colour scheme or specifically to the sidebar text. Example below:

![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/95486289/336eef83-2eda-4768-b05e-9975f25d610f)

Example image from Inkscape. Exported as a standard SVG. Haven't had much success with PS since it'll either be too big in size and resolution or too fuzzy if I reduce it.

![note](https://github.com/FlorianWoelki/obsidian-iconize/assets/95486289/fbb50687-cda5-4aca-9193-10b90a9ca69c)

**Comment by FlorianWoelki** [comment]
I totally agree with you. At the moment, the process is not 100% clear and it's similar to #224. I'll take care of that soon!
